src/app/customer/Customer.tsx

```diff
@@ -370,13 +426,19 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps, Cust
 
         onChangeViewType(CustomerViewType.Login);
     };
+
+    private getCustomCustomerFlowProviderId(paymentMethods: PaymentMethod[] = []) {
+        const paymentProvidersWithCustomFlow = paymentMethods.filter(paymentMethod => paymentMethod?.initializationData?.customContinueFlowEnabled);
+
+        return paymentProvidersWithCustomFlow[0]?.id;
```

Comment:
If there are more than one payment method with `customContinueFlowEnabled`, you just want the first one? I don't think you can make such an assumption.

Reply:
It's a temporary decision. We think to do some functional to get paymentMethod with high priority, because with the implementation we can trigger just one customContinueFlow...

cc @BC-TKozub @BC-ACherednichenko

Reply:
I think this is an important question to answer. With this UX pattern, I think it only makes sense for one payment method that can take over the customer step. The selection of that one payment method should happen on the backend (based on whatever rule we decide). So I think switching over from a collection to a singular property (https://github.com/bigcommerce/bigcommerce/pull/41748) is a step in the right direction.